src/components/Error/Empty.jsx

```diff
@@ -25,6 +25,8 @@ const EmptyCanvas = ({ type, canUpload, localeKey }) => {
         (localeKey && t(`empty.${localeKey}_text`)) ||
         (canUpload && t('empty.text'))
       }
+      layout={false} //  Please add layout=false in the props of the Empty component, and implement the vertical
```

Comment:
I think we can't just do that. We need t implement the vertical centering, no?

Reply:
I succeed in implementing quickly the fix. Thanks